packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -473,6 +473,7 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
   useEffect(() => {
     const closeSidePane = (): void => {
       setSidePaneRenderer(undefined);
+      setUserSetGalleryLayout('floatingLocalVideo');
```

Comment:
This should be in its own function definition as this is not closing a side pane.

Reply:
I agree with Donald here, as the function that you put this in is specific to "closeSidePane"